src/test/java/com/ericsson/ei/jmespath/test/TestJmesPathInterface.java

```diff
@@ -56,8 +56,8 @@ public void testRunRuleOnEvent() {
         } catch (Exception e) {
             log.error(e.getMessage(), e);
         }
-        String ExtractionRules_test = "{ id : meta.id, type : meta.type, time : meta.time, gav : data.gav, fileInformation : data.fileInformation, buildCommand : data.buildCommand }";
-        JsonNode result = unitUnderTest.runRuleOnEvent(ExtractionRules_test, jsonInput);
+        String extractionRulesTest = "{ id : meta.id, type : meta.type, time : meta.time, gav : data.gav, fileInformation : data.fileInformation, buildCommand : data.buildCommand }";
+        JsonNode result = unitUnderTest.runRuleOnEvent(extractionRulesTest, jsonInput);
         assertEquals(result, output);
```

Comment:
One Improvement: if it fails it would be good to see in log directly what differs between the json objects - > assertEquals(expected, actual, "Execpect JSON content:\n" + expected + "\nActual JSON content:\n" + actual);
Then we can maybe see directly what is failing/missing by looking in the console and test result.